config/default.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+module.exports = {
+    aws: {
+        /**
+         * Default AWS region
+         * Note: Not all services are available in all regions
+         * so this is occasionally overridden
+         */
+        region: 'eu-west-2',
+        cloudfrontDistributions: {
+            test: {
+                distributionId: 'E3D5QJTWAG3GDP',
+                origins: {
+                    legacy: 'LEGACY',
+                    newSite: 'ELB-TEST'
+                }
+            },
+            live: {
+                distributionId: 'E2WYWBLMWIN5U1',
+                origins: {
+                    legacy: 'LEGACY',
+                    newSite: 'ELB_LIVE'
+                }
+            }
+        }
+    },
+
+    /**
+     * Domain config
+     * Define current and legacy domain configs,
+     * different variants needed to support domain move
+     */
+    domains: {
+        // Which domain is the primary/canonical domain
+        // @TODO: Update when moving primary domain
+        primary: 'www.biglotteryfund.org.uk',
+        // Which domains should be indexable by search engines
+        // @TODO: Add new domain when moving primary domain
+        indexable: ['www.biglotteryfund.org.uk'],
+        // Explicit config for new domain
+        new: 'www.tnlcommunityfund.org.uk'
+    },
+
+    /**
+     * Common date format strings
+     * @see http://momentjs.com/docs/#/displaying/format/
+     */
+    dateFormats: {
+        month: 'MMMM YYYY',
+        short: 'D MMMM, YYYY',
+        full: 'dddd D MMMM YYYY',
+        fullTimestamp: 'dddd D MMM YYYY (hh:mm a)'
+    },
+    i18n: {
+        urlPrefix: {
+            cy: '/welsh'
+        }
+    },
+    features: {
+        enableDigitalFundApplications: true,
+        enablePrompt: false,
+        enableSurvey: true,
+        enableAbTests: true,
+        enableHotjar: false,
+        useRemoteAssets: true,
```

Comment:
... wonder if we should rename this to `enableRemoteAssets` for the sake of consistency/OCD.